readme.md

```diff
@@ -1,3 +1,3 @@
 # Repository for class project
 
-This readme is a placeholder that should be replaced
+This is for Assignment 1
```

Comment:
[You'll want to make sure your readme contains build instructions](https://github.com/MissouriState-CSC515-Spring2020/advanced-internet-programming/blob/master/project/checkpoint1.md#requirements)